Index: pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-output-headphones.conf
===================================================================
--- pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478.orig/src/modules/alsa/mixer/paths/analog-output-headphones.conf	2010-03-26 12:51:15.000000000 -0700
+++ pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-output-headphones.conf	2010-03-26 13:22:26.000000000 -0700
@@ -51,6 +51,9 @@
 override-map.1 = all
 override-map.2 = all-left,all-right
 
+[Element Line HP Swap]
+switch = on
+
 ; This profile path is intended to control the first headphones, not
 ; the second headphones. But it should not hurt if we leave the second
 ; headphone jack enabled nonetheless.
Index: pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-output.conf
===================================================================
--- pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478.orig/src/modules/alsa/mixer/paths/analog-output.conf	2010-03-26 12:51:15.000000000 -0700
+++ pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-output.conf	2010-03-26 14:08:57.000000000 -0700
@@ -44,6 +44,9 @@
 switch = off
 volume = off
 
+[Element Line HP Swap]
+switch = off
+
 ; This profile path is intended to control the default output, not the
 ; headphones. But it should not hurt if we leave the headphone jack
 ; enabled nonetheless.
