Index: pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-input.conf.common
===================================================================
--- pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478.orig/src/modules/alsa/mixer/paths/analog-input.conf.common	2010-03-10 18:53:09.000000000 -0500
+++ pulseaudio-0.9.22~0.9.21+stable-queue-32-g8478/src/modules/alsa/mixer/paths/analog-input.conf.common	2010-03-10 18:53:28.000000000 -0500
@@ -62,6 +62,10 @@
 [Element Input Source]
 enumeration = select
 
+[Option Input Source:Digital Mic]
+name = input-microphone
+priority = 20
+
 [Option Input Source:Mic]
 name = input-microphone
 priority = 20
